bed/modify.go

```diff
@@ -84,6 +84,9 @@ func MergeHighMem(records []Bed, mergeAdjacent bool) []Bed {
 				currentMax.Score = records[i].Score
 			}
 			currentMax.ChromEnd = numbers.Max(records[i].ChromEnd, currentMax.ChromEnd)
+			if keepAllNames && records[i].Name != "" || keepAllNames && currentMax.Name != "" {
```

Comment:
Does this work when currentMax.Name is blank?  It seems like we would then end up with extra commas at that start.  I think we want to something like:
if keepAllNames && records[i].Name != "" { // do we have a name to add
   if currentMax.Name != "" { // is there already a name there to add on to
      currentMax.Name = currentMax.Name + "," + records[i].Name
   } else { // there is no name and this will be the first one
      currentMax.Name = records[i].Name
   }
}

Reply:
good catch, I was only using this with files that absolutely all had names